scholarx/archive/styles.css

```diff
@@ -29,25 +30,50 @@ hr {
   line-height: 1.7;
 }
 
-#profile-container {
+.mentors-table {
     height: 850px;
     overflow-y: auto;
     margin-top: 20px;
     border-radius: 5px;
     border: 1px solid #eeeeee;
 }
 
-#profile-container table {
-    margin: 5px;
+.mentors-table table {
+  margin: 5px !important;
+}
+
+.mentors-table table th {
+  text-align: center !important;
+}
+
+.mentors-table td,
+.mentors-table th {
+  border-top: 1px solid #eeeeee;
+  text-align: left;
+  padding: 8px;
 }
 
-#profile-container table th {
-    text-align: center;
+.mentees-table {
+  height: 850px;
+  overflow-y: auto;
+  margin-top: 20px;
+  border-radius: 5px;
+  border: 1px solid #eeeeee;
+  display: none;
 }
 
-#profile-container td,
-#profile-container th {
-    border-top: 1px solid #eeeeee;
-    text-align: left;
-    padding: 8px;
+.mentees-table table {
+  margin: 5px;
 }
+
+.mentees-table table th {
+  text-align: center;
+}
+
+.mentees-table td,
+.mentees-table th {
+  border-top: 1px solid #eeeeee;
+  text-align: left;
+  padding: 8px;
+}
+
```

Comment:
This is an extra line right? Remove it @janithlahirukariyawasam

Reply:
```suggestion
```

Reply:
@anjula-sack @YoshithaRathnayake Can you check now guys?

Reply:
Nice